Fix LCD output and add translations

3d-gussner opened the pull request.

3d-gussner committed:
Fix LCD output
Configuration->HW Setup->Checks showed `a`

MSG_FILAMENT `Filament` can only be 8 chars long and only ES and IT had `Filemanto` as translation.
But as ES the `o` in `Filamento` was overwritten by `[Estricto]` I removed the MSG_FILAMENT to be translated.
Saves few bytes.

3d-gussner committed:
Add missing translations
Used `trans -b :cs+de+es+fr+hr+hu+it+nl+no+pl+ro+sk+sv "<text>"` to translate
Looked at pre-translated text to reuse as much as possible.

3d-gussner commented:
Please verify and approve the new translations. Remember that the message is limited to 20 characters and 3 rows,
- [x] :czech_republic:  @DRracer
- [X] :de: @3d-gussner
- [x] :es: 
- [x] :fr: @Ilovemyhous @ErwanAliasr1
- [x] :it: @wavexx
- [x] :poland: @arekm
- [x] :netherlands: @vintagepc @stelgenhof
- [x] :romania: @leptun @Hauzman
- [x] :hungary: @AttilaSVK @Hauzman
- [ ] :croatia: @Prime1910 @PRPA041
- [x] :slovakia: @ingbrzy @shatter136
- [ ] :sweden: @Painkiller56
- [ ] :norway: @OS-kar @trondkla

ErwanAliasr1 reviewed (approved):
FR is :ok:

arekm reviewed (approved):
PL is OK.

Ro3Deee commented:
@leptun, for Romanian:
Instead "Nu există filament încărcat" I'd use "Filamentul nu este încărcat" sau "Filamentul nu este detectat"

ingbrzy commented on lang/po/Firmware_sk.po:
```diff
@@ -2597,6 +2597,16 @@ msgstr "Tryska je horúca! Počkajte na vychladnutie."
+#. MSG_MISSING_FILAMENT_CONTINUE c=20 r=3
+#: ../../Firmware/messages.cpp:157 ../../Firmware/util.cpp:388
+msgid "There is no filament loaded. Continue?"
+msgstr "Nie je vložené žiadne vlákno. Pokračovať?"
```
"Nie je zavedený žiaden filament. Pokračovať?"

ingbrzy replied:
@3d-gussner can you update it?
Thank you

ingbrzy commented on lang/po/Firmware_sk.po:
```diff
@@ -2597,6 +2597,16 @@ msgstr "Tryska je horúca! Počkajte na vychladnutie."
+#. MSG_MISSING_FILAMENT_CANCELLED c=20 r=8
+#: ../../Firmware/messages.cpp:158 ../../Firmware/util.cpp:389
+msgid "There is no filament loaded. Print cancelled."
+msgstr "Nie je vložené žiadne vlákno. Tlač zrušená."
```
"Nie je zavedený žiaden filament. Tlač zrušená."

3d-gussner commented:
> @leptun, for Romanian: Instead "Nu există filament încărcat" I'd use "Filamentul nu este încărcat" sau "Filamentul nu este detectat"

Thanks @petrubecheru @Hauzman. Which is the preferred one?
1. Filamentul nu este încărcat :tada: 
2. Filamentul nu este detectat :heart:

3d-gussner committed:
Update Romainian translations
Thanks to @petrubecheru and @Hauzman

stelgenhof commented on lang/po/Firmware_nl.po:
```diff
@@ -2624,6 +2617,16 @@ msgstr "Mondstuk is heet! Wacht op afkoeling."
+#. MSG_MISSING_FILAMENT_CANCELLED c=20 r=8
+#: ../../Firmware/messages.cpp:158 ../../Firmware/util.cpp:389
+msgid "There is no filament loaded. Print cancelled."
+msgstr "Geen filament geladen. Afdrukken geannuleerd."
```
Although 'afdrukken' is the correct translation for 'to print', in the context of 3D printing it is a bit strange. I suggest to keep the English verb "to print" as that is more commonly used in the community.

So the translation should then be "Geen filament geladen. Printen geannuleerd."

With that said, other translations in this file should be adjusted accordingly for consistency's sake.

3d-gussner committed:
Update Dutch translation
Thanks to @stelgenhof

3d-gussner committed:
Another Dutch translation fix

gudnimg reviewed (approved):
Looks good to me. :) I’ll wait a bit for translators to review before merging.

AttilaSVK commented:
Hungarian should be: "Nincs befűzve filament. Folytassam?" and "Nincs befűzve filament. Nyomtatás megállítva."

3d-gussner committed:
Update Hungarian translation
Thanks to @AttilaSVK

DRracer commented on lang/po/Firmware_cs.po:
```diff
@@ -2599,6 +2592,16 @@ msgstr "Tryska je horká! Počkejte na vychladnutí."
+#. MSG_MISSING_FILAMENT_CONTINUE c=20 r=3
+#: ../../Firmware/messages.cpp:157 ../../Firmware/util.cpp:388
+msgid "There is no filament loaded. Continue?"
+msgstr "Není vloženo žádné vlákno. Pokračovat?"
```
Není vložen filament. Pokračovat?

DRracer commented on lang/po/Firmware_cs.po:
```diff
@@ -2599,6 +2592,16 @@ msgstr "Tryska je horká! Počkejte na vychladnutí."
+#. MSG_MISSING_FILAMENT_CANCELLED c=20 r=8
+#: ../../Firmware/messages.cpp:158 ../../Firmware/util.cpp:389
+msgid "There is no filament loaded. Print cancelled."
+msgstr "Není vloženo žádné vlákno. Tisk zrušen."
```
Není vložen filament. Tisk zrušen.

wavexx commented on lang/po/Firmware_it.po:
```diff
@@ -2622,6 +2615,16 @@ msgstr "L'ugello è caldo! Attendere il raffreddamento."
+#: ../../Firmware/messages.cpp:158 ../../Firmware/util.cpp:389
+msgid "There is no filament loaded. Print cancelled."
+msgstr "Non è caricato alcun filamento. Stampa annullata."
+
```
Looking good! You could marginally shorten the message with `Nessun filamento caricato.` as the first sentence if that's beneficial.

3d-gussner committed:
Update Czech and Italian translations
Thanks to @DRRacer and @wavexx

3d-gussner commented:
Hi
:croatia: @Prime1910 @PRPA041
:slovakia: @ingbrzy @shatter136
:sweden: @Painkiller56
:norway: @OS-kar @trondkla

Can you please let me know if you will be able to review the changes until end of this week? Also please let me know if you are still interested in reviewing the community translations. If not, I will not ask you anymore and will NOT translate new messages anymore or use "faulty" computer based translations instead.

shatter136 reviewed (commented):
Hi. Sorry. In _sk, I would rather write "Nie je vložený filament. ... "
And please keep us still in the loop. Thank you.

3d-gussner committed:
Update Slovak translation
Thanks to @ingbrzy @shatter136

3d-gussner committed:
Update Community translation list
- Croatian, Swedish and Norwegian are inactive since the last few translation review requests have not been done
